packages/manager/modules/bm-server-components/src/ipmi/order-kvm/controller.js

```diff
@@ -13,7 +13,9 @@ export default class BmServerComponentsOrderKvmController {
   $onInit() {
     this.serviceName = this.server.name;
     const [datacenter] = this.server.datacenter.split('_');
-    this.datacenter = datacenter.toLowerCase();
+    this.datacenter = this.IpmiService.constructor.getRealDCAgoraName(
+      datacenter.toLowerCase(),
+    );
```

Comment:
No need to define the mapping here, as it concerns kvm order, it should be done just before calling the API, therefore directly in `IpmiService`. By the way, the mapping function is alredy defined in the service so it sounds more logical